set.mm

```diff
@@ -481233,6 +481329,394 @@ such that every prime ideal contains a prime element (this is a
+        rspectopn syl6reqr nfcv nfrab1 wrex notrab eqeq2i ssrab2 elpwi ssdifsym
+        cpw nfv a1i syl2anc eqcom bitr3id ad2antlr rexbidva cprmidl fvexi rabex
+        elrnmpti syl6bbr pm5.32da wral elpw2 mpbir rgenw rnmptss ax-mp pm4.71ri
+        syl6bb sseli cvv vex elrnmpt anbi2i bitri rabid 3bitr4g eqrd eqtrd ) BU
+        AMZFDBUBUCZDNZENOZUDZEAPZUEZUFZAHNZUGZGUFMZHAUSZPZWKWRCUHUCFABCDEWLWRIW
+        LQKWRQUIJUJWKHWRXCWKHUTHWRUKXAHXBULWKWSXBMZWSWPRZDWLUMZSZXDXASWSWRMZWSX
+        CMWKXDXFXAWKXDSZXFWTWNEAPZRZDWLUMXAXIXEXKDWLXDXEXKTWKWMWLMXEWSAXJUGZRZX
+        DXKXLWPWSWNEAUNUOXDXMXJWTRZXKXDXJAOZWSAOXMXNTXOXDWNEAUPVAWSAUQXJWSAURVB
+        XJWTVCVTVDVEVFDWLXJWTGLWNEAABVGKVHZVIVJVKVLXHXDXHSXGXHXDWRXBWSWPXBMZDWL
+        VMWRXBOXQDWLXQWPAOWOEAUPWPAXPVNVOVPDWLWPXBWQWQQZVQVRWAVSXHXFXDWSWBMXHXF
+        THWCDWLWPWSWQWBXRWDVRWEWFXAHXBWGWHWIWJ $.
+
+      $d P x y $.  $d P z $.  $d R i s x y $.  $d R i z $.  $d R j $.
+      $d V i s x y $.  $d V i z $.
+      $( The Zariski topology is a topology, and its closed sets are images by
+         ` V ` of the ideals of ` R ` .  (Contributed by Thierry Arnoux,
+         16-Jun-2024.) $)
+      zartopn $p |- ( R e. CRing -> ( J e. ( TopOn ` P ) /\ ran V = ( Clsd ` J
+        ) ) ) $=
+        ( vs wcel cfv ccld wceq wa cv crab eqid vx ccrg ctopon crn cdif cpw wss
+        vy vz clidl wral ssrab2 cprmidl fvexi elpw2 mpbir rgenw rnmptss a1i crg
+        ax-mp crngring c0g csn cmpt rabeqi mpteq2i eqtri zarcls0 wfun cdm lidl0
+        funmpt2 rabex dmmpti eleqtrrdi fvelrn sylancr eqeltrrd zarclsint ismred
+        syl cbs c0 lidl1 eleqtrdi zarcls1 mpbiri mpdan syl2anc zarclsun mretopd
+        zarcls eleq1d fveq2d eqeq2d anbi12d mpbird ) BUBMZFAUCNZMZGUDZFONZPZQAL
+        RUEXBMLAUFZSZWTMZXBXFONZPZQWSUAUHLAXFXBWSXBAUIXBXEUGZWSDRERUGZEASZXEMZD
+        BUJNZUKXJXMDXNXMXLAUGXKEAULXLAABUMJUNZUOUPUQDXNXLXEGKURVAUSWSBUTMZAXBMB
+        VBZXPBVCNZVDZGNZAXBABDEGXRGDXNXLVEDXNXKEBUMNZSZVEKDXNXLYBXKEAYAJVFVGVHZ
+        JXRTZVIXPGVJZXSGVKZMXTXBMDXNXLGKVMZXPXSXNYFBXNXRXNTYDVLDXNXLGXKEAXOVNKV
+        OZVPXSGVQVRVSWBBUIRDEGYCVTWAWSBWCNZGNZWDXBWSYIXNMZYJWDPZWSYIYFXNWSXPYIY
+        FMZXQYIBYFYHYITZWEWBZYHWFWSYKQYLYIYIPYNYIBDEYIGYCYNWGWHWIWSYEYMYJXBMYEW
+        SYGUSYOYIGVQWJVSBDEGUARUHRYCWKXFTWLWSXAXGXDXIWSFXFWTWSXPFXFPXQABCDEFGLH
+        IJKWMWBZWNWSXCXHXBWSFXFOYPWOWPWQWR $.
+    $}
+
+    $d R i j k $.
+    $( The Zariski topology is a topology.  (Contributed by Thierry Arnoux,
+       16-Jun-2024.) $)
+    zartop $p |- ( R e. CRing -> J e. Top ) $=
+      ( vi vj vk ccrg wcel cprmidl cfv ctopon ctop clidl cv wss crab cmpt wceq
+      crn ccld eqid sseq1 rabbidv cbvmptv zartopn simpld topontop syl ) AIJZCAK
+      LZMLJZCNJUKUMFAOLZFPZGPZQZGULRZSZUACUBLTULABHGCUSDEULUCFHUNURHPZUPQZGULRU
+      OUTTUQVAGULUOUTUPUDUEUFUGUHULCUIUJ $.
+
+    ${
+      $d P i j k $.
+      zartop.3 $e |- P = ( PrmIdeal ` R ) $.
+      $( The base set of the Zariski topology.  (Contributed by Thierry Arnoux,
```

Comment:
What is ment by "base set" here?

Reply:
This is meant in the same sense as in [~ tpsuni](https://us.metamath.org/mpeuni/tpsuni.html) : the underlying "set of points" are prime ideals.

The comment is modelled after e.g. [~xkotopon](https://us.metamath.org/mpeuni/xkotopon.html).

Reply:
I've updated this comment, with:
> The points of the Zariski topology are the prime ideals.